full-stack-tests/rpc-interface/src/frontend/IModelConnection.test.ts

```diff
@@ -60,136 +60,6 @@ describe("IModel Connection", () => {
   });
 });
 
-describe("IModelConnection Tiles", () => {

```

Comment:
@pmconne I'll assume we can follow up with a new set of tests that cover the individual RPCs separately? I have an action item to ensure we're covering all rpc endpoints again.